swap_meet/vendor.py

```diff
@@ -1,2 +1,84 @@
+import uuid
+
+#from item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=[]):
+        self.inventory = inventory #each vendor will have an attribute named :inventory (an empty list)
+        
+    def add(self, item):
+        self.inventory.append(item)
+        print(self.inventory)
+        print(item)
+        return item 
+    
+    def remove(self, item): 
+        
+        if item not in self.inventory:
+            return False
+        else:
+            self.inventory.remove(item)
+            return item
```

Comment:
[suggestion] If your `if` body has a `return`, you *technically* don't need the `else:`. You could do:

```python
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return item
```